src/components/rangeUsePlanPage/conditions/index.js

```diff
@@ -5,21 +5,32 @@ import { CONDITIONS } from '../../../constants/fields'
 import * as strings from '../../../constants/strings'
 import PermissionsField from '../../common/PermissionsField'
 import { TextArea } from 'formik-semantic-ui'
+import { useUser } from '../../../providers/UserProvider'
 
 const Conditions = ({ plan }) => {
```

Comment:
We should probably hide this component for AHs completely in `PlanForm` when no conditions are provided.